### PR TITLE
A way to check backend i18n

### PR DESCRIPTION
**Pull request by camsaul**
Add totally sweet `lein check-i18n` command that reverses all backend strings marked for i18n.

screen shot 2018-01-31 at 1 27 40 pm

### CODE REVIEW

**Comment by salsakran** [comment]
looks awesome.

**Comment by salsakran** [comment]
linting fails however.

**Comment by camsaul** [comment]
@salsakran fixed

**Review by salsakran — APPROVED** [review]

File: src/metabase/core.clj [hunk 1]
```diff
@@ -2,9 +2,7 @@
 (ns metabase.core
   (:gen-class)
   (:require [cheshire.core :as json]
-            [clojure
-             [pprint :as pprint]
-             [string :as s]]
+            [clojure.pprint :as pprint]
             [clojure.tools.logging :as log]
             environ.core
```
Comment (senior) [inline src/metabase/core.clj]: I didn't see any references in core to `env`. Can we remove this?

**Review by senior — APPROVED** [review]
:+1:

File: src/metabase/core.clj (hunk 1 above)
Comment (camsaul) [reply src/metabase/core.clj]: done

**Pull request merged**